sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/BaseAuthorizationTokenProvider.java

```diff
@@ -49,6 +49,7 @@ private static String getResourceSegment(ResourceType resourceType) {
         case Document:
             return Paths.DOCUMENTS_PATH_SEGMENT;
         case DocumentCollection:
```

Comment:
could you please check on the other auth modes as well? AAD, is this compatible with AAD as well?
@milismsft FYI